.rubocop.yml

```diff
@@ -0,0 +1,58 @@
+AllCops:
+  Exclude:
+    - "README.md"
+    - "Guardfile"
+    - "Rakefile"
+
+  DisplayCopNames: true
+
+Layout/LineLength:
+  Max: 120
+Metrics/MethodLength:
+  Max: 20
+Metrics/AbcSize:
+  Max: 50
+Metrics/ClassLength:
+  Max: 150
+Metrics/CyclomaticComplexity:
+  Max: 100
+Metrics/PerceivedComplexity:
+  Max: 100
+Metrics/BlockLength:
+  ExcludedMethods: ['describe']
+  Max: 30
```

Comment:
Linters config: Do not make any changes in config files. These lines are modified.